src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.ts

```diff
@@ -75,6 +75,13 @@ export class InstructorCourseEditPageComponent implements OnInit {
     creationTimestamp: 0,
     deletionTimestamp: 0,
   };
+  savedCourse: Course = {
```

Comment:
`originalCourse`?

`savedCourse` is more like it is saved. Sounds a bit weird.